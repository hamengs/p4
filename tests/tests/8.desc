Call free, observe that it doesn't call deallocuvm
