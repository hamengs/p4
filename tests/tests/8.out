XV6_TEST_OUTPUT Start of program - base pages:3 huge pages:0
XV6_TEST_OUTPUT After first vmalloc huge, 4 MB - program base pages:3 huge pages:2
XV6_TEST_OUTPUT After second vmalloc, 100 bytes - program base pages:11 huge pages:2
XV6_TEST_OUTPUT After third vmalloc, 5000 bytes - program base pages:11 huge pages:2
XV6_TEST_OUTPUT Freeing vmalloc allocations
XV6_TEST_OUTPUT After freeing data - program base pages:11 huge pages:2
